client/src/components/StreamField/blocks/StructBlock.js

```diff
@@ -113,6 +113,19 @@ export class StructBlock {
     return value;
   }
 
+  getTextLabel(opts) {
+    /* Use the text label of the first child block to return one */
```

Comment:
Idea: Might be nice to allow a developer to specify a field name to use?

Reply:
Sure - there's a lot of scope for making this more customisable, perhaps with some limited templating like `"${first_name} ${last_name}"`. This will do as a first pass though!